types/eslint-ast-utils/index.d.ts

```diff
@@ -0,0 +1,155 @@
+// Type definitions for eslint-ast-utils 1.1
+// Project: https://github.com/jfmengels/eslint-ast-utils#readme
+// Definitions by: Mark McCann <https://github.com/marksmccann>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
This is still an old-style package; the CI apparently don't catch this yet (oops) but you will need a `package.json`. The new dts-gen release should generate this properly, or you can model another package.